### PR TITLE
Add large scale image classification tutorial to tutorials index. Also fix some formatting issues in the tutorial.

### PR DESCRIPTION
**Pull request by indhub**
@mli, @zackchase, @madjam, @nswamy, @Roshrini, @jiajiechen
Please review and merge.

### CODE REVIEW

File: docs/tutorials/index.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ These tutorials introduce fundamental concepts in deep learning and their realiz
    basic/symbol
    basic/module
    basic/data
+   vision/large_scale_classification
```
Comment (madjam) [inline docs/tutorials/index.md]: Why is this under Basics and not under Computer Vision?
In fact we also need to add linear regression to the tutorial index.
Can you rename Computer Vision to "MXNet: Training and Inference"

**Comment by mli** [comment]
what're changed? git diff shows every line is updated, but i didn't see too many changes if reading the content.

**Comment by indhub** [comment]
@mli Check the individual commits.

File: docs/tutorials/index.md [hunk 2]
```diff
@@ -17,14 +17,15 @@ These tutorials introduce fundamental concepts in deep learning and their realiz
    basic/data
 ```
 
-### Computer Vision
+### MXNet: Training and Inference
```
Comment (piiswrong) [inline docs/tutorials/index.md]: ?

Comment (indhub) [inline docs/tutorials/index.md]: That was requested by @madjam

Comment (madjam) [inline docs/tutorials/index.md]: @piiswrong We need to find a place for Linear Regression and I didn't think **Basics** was the right place for it. That is way I suggested renaming this. Later when we add more tutorials, we can revive Computer Vision section.

Do you have concerns?

Comment (piiswrong) [inline docs/tutorials/index.md]: `MXNet: ` seems superfluous?

**Pull request merged**